src/utils/helpers.ts

```diff
@@ -78,11 +78,14 @@ export function formatSearchQueryString(
 }
 
 export async function getHtmlUrl(url: string, token: string): Promise<string> {
-  const response: Issue | IssueComments | PullRequest = (
-    await apiRequestAuth(url, 'GET', token)
-  ).data;
-
-  return response.html_url;
+  try {
+    const response: Issue | IssueComments | PullRequest = (
+      await apiRequestAuth(url, 'GET', token)
+    ).data;
+    return response.html_url;
+  } catch (err) {
+    console.error('Failed to get html url');
```

Comment:
Confirming this is intentionally still there?

Reply:
Yes, intentional for now.

Reply:
I see 👀 
You are planning on having a central place where you define all error messages aren't you? 👀

Reply:
Do you guys want me to remove the try/catch from subject/helper apiRequestAuth calls?